.travis.yml

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright © 2016 Cask Data, Inc.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+language: java
+
+jdk:
+  - oraclejdk7
```

Comment:
Cool. This would be good as we currently only run unit tests using one version of Java (Oracle JDK 7), so more coverage here would be good. We do run integration tests on other Java versions, just not unit tests.